libmachine/mcnutils/b2d.go

```diff
@@ -304,10 +306,12 @@ func (b *b2dISO) version() (string, error) {
 		return "", err
 	}
 
-	verRegex := regexp.MustCompile(`v\d+\.\d+\.\d+`)
-	ver := string(verRegex.Find(isoMetadata))
-	log.Debug("local Boot2Docker ISO version: ", ver)
-	return ver, nil
+	// Original magic file string looks similar to this: "Boot2Docker-v0.1.0              "
+	// This will return "v0.1.0" given the above string
+	vers := strings.TrimSpace(string(isoMetadata))[prefixOffset:]
```

Comment:
> Would IndexOf "-v" or something be more resilient?

Yeah, actually I like that, I had considered expanding the existing regex but I think I like that better.